packages/spatial-id-viewer/src/views/weather/view/hooks/load-models.tsx

```diff
@@ -19,17 +19,17 @@ import { mapGetOrSet } from '#app/utils/map-get-or-set';
 
 interface WeatherInfo extends Record<string, unknown> {
   id: string;
-  spatialId: string;
-  startTime: string;
-  endTime: string;
-  windDirection: number;
-  windSpeed: number;
-  cloudRate: number;
-  temperature?: number;
-  dewPoint?: number;
-  pressure?: number;
-  precipitation: number;
-  visibility?: number;
+  spatialId?: string;
+  startTime?: string;
+  endTime?: string;
+  'windDirection (degree)': number;
```

Comment:
I hope its not recommend to name vairables in this manner. better to rename it to windDirectionDegree

Reply:
fix in the below other variables as well